docs/google-kubernetes-tutorial.md

```diff
@@ -154,6 +154,18 @@ Now the number of TiKV pods is increased from the default 3 to 5. You can check
 
 	kubectl get po -n tidb
 
+## Destroy the TiDB cluster
+
+When the TiDB cluster is not needed, you can delete with the following command:
+
+	helm delete tidb --purge &&
+	kubectl delete
+
+The above commands only delete the running pods, the data is persistent. If you do not need the data anymore, you should run the following command to clean the data and the dynamically created persistent disks:
+
+	kubectl delete pvc -n tidb -l app.kubernetes.io/instance=tidb,app.kubernetes.io/managed-by=tidb-operator &&
```

Comment:
This is a dangerous operation, give use a warning?

Reply:
Since it's just a tutorial, and the statement above the command already said it will clean the data and persistent disks. So not a big problem I think.